dvc/command/data_sync.py

```diff
@@ -207,6 +215,12 @@ def add_parser(subparsers, _parent_parser):
         default=False,
         help="Push cache for subdirectories of specified directory.",
     )
+    push_parser.add_argument(
+        "--drop-index",
+        action="store_true",
+        default=False,
+        help="Drop local index for the specified remote cache.",
```

Comment:
same here - let's not use `cache` .. @jorgeorpinel could please review messages, btw?

Reply:
Agree. We prefer "remote storage" or just "remote" as of now.